misk-crypto/README.md

```diff
@@ -17,7 +17,7 @@ In one of your app's module files:
 ```$kotlin
 class MyAppModule : KAbstractModule {
   override fun configure() {
-    val kmsClient = AwsKmsClient().withDefaultCredentials()
+    install(AwsKmsClientModule()) // will provide an AWS client with default credentials
     install(CryptoModule(kmsClient, cryptoConfig))
```

Comment:
Where's `kmsClient` coming from in L21 now?

Reply:
Good catch, it should've been deleted.

Just to reiterate, the main thing I changed here is that now `CryptoModule` expects a `KmsClient` to be bounded by the user instead of receiving it is a parameter.
The `CryptoModule` will also throw an exception during `configure` if no client was provided.